### ISSUE TITLE
`--refcall` has no effect with polyclone?

### ISSUE THREAD

**Issue by bredelings** [luntergroup/octopus#184]
**Describe the bug**
It looks like the `--refcall` option has no effect with polyclone. 

Am I missing something, or does refcall not work with polyclone?

If it is not there, is there a specific issue (e.g. finding haplotypes for non-variant positions) that is preventing this, or is the information all there and just not printed?

(The context is that I'd like to distinguish between non-variant sites where reads support the reference, and sites where there is no coverage.  So, perhaps using the AD field or the DP field.)

**Version**

```shell
$ octopus --version
octopus v0.7.4 (develop c611a2c9)
```

Command line to run octopus:
```shell
$ octopus -I combined.dedup.bam  -R /home/br51/malaria/reference/plasmo-combined.fasta -o api.poly3.vcf.gz  -T LT635626  --bamout api.poly3.bam --bamout-type FULL  --annotations AD -C polyclone --max-clones 3 --sequence-error-model PCR --refcall --threads 16 
```

**Comment by bredelings** [comment]
It does seem that I could just generate a `--refcall=POSITIONAL` or `--refcall=BLOCKED` VCF file _in addition_ to the polyclone VCF file to get the relevant information.

**Comment by dancooke** [comment]
> It looks like the --refcall option has no effect with polyclone.

Right, the `--refcall` functionality is currently only implemented in the `individual` calling model, and even here I wouldn't claim that it's fully functional - there are some serious performance issues. I'm yet to settle on a model - it's a really difficult problem - hence why it hasn't been implemented in other calling models.

**Comment by bredelings** [comment]
Thanks!  That makes sense.
